public-ui/dispatch.yaml

```diff
@@ -0,0 +1,7 @@
+dispatch:
+  - url: "aou-db-staging.appspot.com/v1/*"
+    service: api
+  - url: "*/favicon.ico"
+    service: default
+  - url: "aou-db-staging.appspot.com/"
+    service: default
```

Comment:
a new github feature let me know that there needs to be a new line at the end of the file.